recipes/dowhy/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "dowhy" %}
+{% set version = "0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 99d58f0098cf37f2b69073f9b47253cafe13f1136d694cd972bec98e9c6ee484
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py<34]
+
+requirements:
+  host:
+    - pip
+    - python
```

If (and only if) it's noarch you can constrain Python using:
```suggestion
    - python >=3.4
```

---

thank you @chrisburr for your super-quick response! I have added all your suggested changes.